packages/devextreme/js/__internal/grids/new/grid_core/selection/const.ts

```diff
@@ -0,0 +1,17 @@
+export enum SelectionMode {
```

Comment:
better not to use enums, only type unions
`export type SelectionMode = 'single' | 'none' ` etc
And better to import existing type from .d.ts